### DISCUSSION TITLE
(suggestion)crit misc : how much damage to deal before crit

### DISCUSSION THREAD

**Discussion by JDNA364PROC** [Fedoraware/Fedoraware#980] [Ideas]
we have those crit indicators but it would be cool to see how much damage we need to deal before we actually get a crit (1000/1000 crit bucket no crit)

once again i dont force you to do that , your choice

**Comment by vaitomanocucpp** [comment]
ok

**Comment by N1kolasFlaV** [comment]
too hard, u cant find it on uc or here.....

**Comment by ghost** [comment]
Well, even cathook have it

**Comment by ghost** [comment]
we know how to implement it.

**Comment by ghost** [comment]
> we know how to implement it.

Great, when adding it? Lol

**Comment by ghost** [comment]
when someone feels like it or when i rewrite crithack

**Comment by vaitomanocucpp** [comment]
> when someone feels like it or when i rewrite crithack

never lol

**Comment by JDNA364PROC** [comment]
> when someone feels like it or when i rewrite crithack

we dont need a crithack rework , we need a working crit indicator.

**Comment by johnathon-walnut** [comment]
I wonder why the crit indicator *isn't* working :)

> > when someone feels like it or when i rewrite crithack
> 
> we dont need a crithack rework , we need a working crit indicator.

**Comment by JDNA364PROC** [comment]
> I wonder why the crit indicator _isn't_ working :)
> 
> > > when someone feels like it or when i rewrite crithack
> > 
> > 
> > we dont need a crithack rework , we need a working crit indicator.

ur too lazy to fix this
